libs/sdk/testing/package.json

```diff
@@ -20,7 +20,7 @@
     "@angular/core": "^15.2.9",
     "@angular/platform-browser": "^15.2.9",
     "@skyux/i18n": "0.0.0-PLACEHOLDER",
-    "axe-core": "^3.5.6 || ^4.6.3"
+    "axe-core": "^3.5.6 || ^4.6.3 || ^4.7.2"
```

Comment:
I looked at the changelog and didn't see any breaking changes between `4.6` and `4.7`. At any rate, `^4.6.3` evaluates to the same SemVer range as `^4.7.2` so we can probably just remove `^4.6.3` and require folks to use the latest version of `4`.

Reply:
Nothing in the changelog suggests a breaking change, but the update does cause passing tests to potentially fail. I'll add it to the flex time agenda.

Reply:
We decided to leave the dependency at `4.6` but consumers using `4.7` should no longer see errors.